homeassistant/components/portainer/config_flow.py

```diff
@@ -31,6 +32,21 @@
     }
 )
 
+STEP_RECONF_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_API_TOKEN): str,
+        vol.Required(SECTION_DANGER_ZONE): data_entry_flow.section(
```

Comment:
Not sure why we call this danger zone. While it's a cool name that github for example uses, I don't think we should use that here